xbmc/GUIInfoManager.cpp

```diff
@@ -5320,6 +5336,14 @@ int CGUIInfoManager::TranslateSingleString(const std::string &strCondition, bool
           return videoplayer[i].val;
       }
     }
+    else if (cat.name == "player_process")
+    {
+      for (size_t i = 0; i < sizeof(player_process) / sizeof(infomap); i++)
+      {
+        if (prop.name == player_process[i].str)
```

Comment:
Perhaps prop.name here needs to be changed to lower case to allow it to be case insensitive as mentioned here: [http://forum.kodi.tv/showthread.php?tid=285695&pid=2390071#pid2390071](url)